.env.sample

```diff
@@ -27,6 +27,7 @@ HOSTNAME=localhost
 KINTO_HOST=http://kinto:8888/v1
 KINTO_PASS=experimenter
 KINTO_USER=experimenter
+KINTO_ADMIN_URL=http://localhost:8888/v1/admin
```

Comment:
We can derive this in settings like this:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/settings.py#L374

Reply:
Derive from what? I tried KINTO_HOST, but it's using the within-docker URL and not the externally-accessible URL

Reply:
Oh consarnit it's different on local but not on stage/prod.  Well if we make it a separate env var then we've put the remote settings domain in twice, so if we ever go to update it we'll have to remember to do it in both places.  Or we could move the localhost hard coded into settings and then just do something like

```py
KINTO_ADMIN_URL = 'https://localhost:8888/v1/admin' if DEBUG else urljoin(KINTO_HOST, 'admin')
```

Both have their disadvantages.

Reply:
And we'll have to file a bugzilla to deploy the new env var before this lands if we add it.

Reply:
Hmm, if KINTO_HOST is accessible on stage & prod, I could derive it in the default and just set it explicitly for dev.

Reply:
oh hype great thinking 👏 👏 👏 👏 👏  then we just have to be aware that there's an env var in the sample that doesn't apply to stage/prod, but sample doesn't get used on stage/prod so that should probably just work itself out 👍